lstchain/calib/camera/flatfield.py

```diff
@@ -74,6 +79,20 @@ def __init__(self, **kwargs):
         self.arrival_times = None  # arrival time per event in sample
         self.sample_masked_pixels = None  # masked pixels per event in sample
 
+        # declare time calibrator None if MC events
+        if self.time_calibration_path is None:
```

Comment:
This comment is misleading. It says time_calibration is None for mc, but it does not test if it's mc, it tests if a file was given.

Is a timecalibration path required for observed data? If yes, this script should fail if observed data is analyzed and no time calibration path was given.

If it is not required, the comment should just be something like "don't do time calibration if no calibration file was provided"

Reply:
I took out the comment